packer/scripts/Artifacts/MsSql/Structure/Ods/2100-CreateProcedureLoadStudentFallMembership.sql

```diff
@@ -0,0 +1,210 @@
+        AND home.EducationOrganizationId = homespa.EducationOrganizationId
+        AND home.ProgramEducationOrganizationId = homespa.ProgramEducationOrganizationId
+        AND home.ProgramName = homespa.ProgramName
+        AND home.ProgramTypeDescriptorId = homespa.ProgramTypeDescriptorId
+        AND home.StudentUSI = homespa.StudentUSI
+    LEFT JOIN edfi.Descriptor imm WITH (NOLOCK) ON
+        imm.DescriptorId = seodx.StudentCharacteristicDescriptorId
+        AND imm.CodeValue = 'Immigrant'
+    LEFT JOIN (
+        SELECT
+            seodsc.EducationOrganizationId,
+            seodsc.StudentUSI,
+            d.CodeValue,
+            d.Description
+        FROM edfi.StudentEducationOrganizationAssociationStudentCharacteristic seodsc WITH (NOLOCK)
+        INNER JOIN edfi.Descriptor d ON
+            seodsc.StudentCharacteristicDescriptorId = d.DescriptorId
+            AND d.CodeValue = 'Parent in Military'
+        ) parentinmilitary ON
+            seod.StudentUSI = parentinmilitary.StudentUSI
+            AND seod.EducationOrganizationId = parentinmilitary.EducationOrganizationId
+    LEFT JOIN (
+        SELECT
+            seodsc.EducationOrganizationId,
+            seodsc.StudentUSI,
+            d.CodeValue,
+            d.Description
+        FROM edfi.StudentEducationOrganizationAssociationStudentCharacteristic seodsc
+        INNER JOIN edfi.Descriptor d ON
+            seodsc.StudentCharacteristicDescriptorId = d.DescriptorId
+            AND d.CodeValue = 'Single Parent'
+        ) singleparent ON
+            seod.StudentUSI = singleparent.StudentUSI
+            AND seod.EducationOrganizationId = singleparent.EducationOrganizationId
+    LEFT JOIN (
+        SELECT DISTINCT
+            spa.StudentUSI,
+            spa.ProgramEducationOrganizationId
+        FROM edfi.GeneralStudentProgramAssociation spa
+        WHERE spa.ProgramName = 'Special Education'
+            AND (spa.ENDDate >= @October_1st or spa.ENDDate IS NULL)
+            AND spa.BeginDate <= @October_1st) specialeducation ON
+        ssa.StudentUSI = specialeducation.StudentUSI
+        AND school.LocalEducationAgencyId = specialeducation.ProgramEducationOrganizationId
+    LEFT JOIN reporting.GetHomeLanguageCode() homelanguage ON
+        seod.StudentUSI = homelanguage.StudentUSI
+        AND seod.EducationOrganizationId = homelanguage.EducationOrganizationId
+    LEFT JOIN edfi.StudentEducationOrganizationAssociationCohortYear seodcy WITH (NOLOCK) ON
+        seodcy.CohortYearTypeDescriptorId = (SELECT TOP 1 DescriptorId FROM edfi.Descriptor WHERE namespace like '%CohortYearType%' AND CodeValue='Ninth grade')
+        AND seodcy.StudentUSI = seod.StudentUSI
+        AND seodcy.EducationOrganizationId = seod.EducationOrganizationId
+    WHERE
+        ssa.EntryDate <= @October_1st
+        AND (ssa.ExitWithdrawDate >= @October_1st or ssa.ExitWithdrawDate IS NULL)
+        AND msid.total_fte >= (CASE WHEN d.CodeValue IN ('HP', 'PK') THEN 0 else 51 END)
```

Comment:
@jamessmckay codes 'hp' and 'pk' do not exist in the code descriptors. Need to figure out what they represent.

Reply:
@jamessmckay change this to msid.total_fte >= 0.51 (we are dropping the case statement and the note decimal 0.51 instead of 51)

Reply:
done